<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GradientEditor</class>
 <widget class="QWidget" name="GradientEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>227</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QComboBox" name="gradientType">
       <item>
        <property name="text">
         <string>Linear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Radial</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Conical</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="gradientRepeat">
       <item>
        <property name="text">
         <string>No repeat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Repeating</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Mirrored</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GradientStopsEditor" name="gradientEditor" native="true"/>
   </item>
   <item>
    <widget class="QCheckBox" name="editCheckbox">
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="editorPage">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Placeholder; will show a detailed editor pane here.</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="selectionPage">
      <layout class="QHBoxLayout" name="selectionPageLayout">
       <property name="margin">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GradientStopsEditor</class>
   <extends>QWidget</extends>
   <header>GradientStopsEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
